taskcluster/ci/toolchain/kind.yml

```diff
@@ -0,0 +1,57 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+  - taskgraph.transforms.job:transforms
+  - taskgraph.transforms.cached_tasks:transforms
+  - taskgraph.transforms.task:transforms
+
+job-defaults:
+  routes:
+    - notify.email.a-s-ci-failures@mozilla.com.on-failed # TODO only notify for github-push
```

Comment:
It can be achieved this way:

```yaml
  routes:
    by-tasks-for:
        github-push:
            - notify.email.a-s-ci-failures@mozilla.com.on-failed
        default: []
```